meta/send.rb

```diff
@@ -598,3 +598,39 @@
   mutation 'foo(self)'
   mutation 'foo(nil)'
 end
+
+Mutant::Meta::Example.add :send do
+  source 'Array(a)'
+
+  singleton_mutations
+  mutation 'a'
+  mutation s(:send, nil, :Array)
```

Comment:
"normally" using AST input for specifying a migration is a sign of a bug (or an unparser issue).

I've to research the reason here.

Reply:
I am not sure why I put it that way but I have fixed it.

Reply:
I suspect at the time there was an unparser issue, or it was a debugging leftover.